Yoast/yoast-components
# Change slug field to input field instead of draftjs editor

## Summary

This PR can be summarized in the following changelog entry:

* Not applicable

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure the slug inputfield looks and works as expected.
* Use a `%` character in the slug input field to verify it's not a DraftJS editor.

Fixes Yoast/wordpress-seo#9603
